media-gfx/openvdb/openvdb-11.0.0.ebuild

```diff
@@ -0,0 +1,361 @@
+# Copyright 1999-2023 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+PYTHON_COMPAT=( python3_{10..12} )
+
+LLVM_MAX_SLOT=15
```

Comment:
Oh also wanted to ask about this. I'm pretty sure 15 is being removed soon, does it really not support a newer one? :\

Reply:
OpenVDB's documentation [suggests](https://www.openvdb.org/documentation/doxygen/dependencies.html#depDependencyTable) using the latest version of Clang.

Reply:
AX won't compile with anything above llvm-15 see https://github.com/AcademySoftwareFoundation/openvdb/blob/master/openvdb_ax/openvdb_ax/CMakeLists.txt#L116-L120. That's why I originally didn't touch the ax mask. It's also the only place where LLVM_MAX_SLOT is relevant. You can compile OpenVDB itself with any clang.